.github/workflows/tests-thorough.yml

```diff
@@ -30,11 +30,11 @@ jobs:
         # Running tests with the Gradle daemon on windows agents leads to some very strange
         # JVM crashes for some reason. Most likely a problem of Gradle/GitHub/Windows server
         run: >
-          ./gradlew test --stacktrace --no-daemon --no-parallel 
+          ./gradlew test --stacktrace --no-daemon --no-parallel --continue 
```

Comment:
Not entirely sure how adding this flag affects the build - could you please share why it was needed or what it improves?

Reply:
Without `--continue` Gradle will stop when the first task fails. With `--continue` Gradle will keep running tasks (if possible). Continuing even if one task fails is useful when running all tests because it helps to determine if your change has broken just one test vs many tests.

Reply:
It's indeed useful :+1:  For some reason I thought that it was the default behaviour, and that adding this flag had something to do with the rest of the changes, but perhaps it's just an unrelated minor improvement